### DISCUSSION TITLE
USB Drives?

### DISCUSSION THREAD

**Discussion by EpicLPer** [007revad/Synology_HDD_db#113] [General]
Heya!

I think I already know that the answer is no to this, but is there any way to make a volume out of USB drives by adding them to the database? I'd love to use HyperBackup with those as sources, not as targets, and to be able to check the storage status way easier that way.

Thanks!

**Comment by 007revad** [comment]
The first few version of this script unintentionally added USB drives the database. But I doubt DSM would have let anyone create a storage and volume on them because DSM checks if drives are external or not. Apparently it used to possible in DSM 6 by editing a file to make DSM think they were internal drives, but DSM 7 removed that.

Have a look at Synology's USB Copy package. It can import data as mulit-version, mirrored or incremental. And you can set it to run the backup whenever you plug in the USB drive (it ejects the USB drive when it's finished).
https://kb.synology.com/en-us/DSM/help/USBCopy/usbcopy_general?version=7

**Reply by EpicLPer** [reply]
Hm, sadly this wouldn't be useful for me then :( I use the external drives almost exclusively for Plex, since so far it wouldn't matter too much in case they die. Making a volume, even if unreliable (in the sense of a proper NAS lol) would still be better to have Hyper Backup. A copy job from external to internal isn't going to work since I have 8TB external, and only like 3TB free internal.
